### PR TITLE
AI

### PR DESCRIPTION
**Pull request by Westlanderz**
This PR adds more randomness to the movement of the AI units and lets them select a random tile in their range instead of the most left tile.

### CODE REVIEW

**Review by drfdaniel — DISMISSED** [review]
Looks good to me

File: Assets/Scripts/Core/Controllers/GridCombatSystem.cs [hunk 1]
```diff
@@ -119,6 +119,9 @@ private enum MouseButtons {
 				GameEnded?.Invoke(this, false);
 			} else {
 				redTeamList_.Remove(unit);
+				if(redTeamList_.Count <= 0)	{
```
Comment (Dennisbonke) [inline Assets/Scripts/Core/Controllers/GridCombatSystem.cs:122]: Seems like an erroneous tab was inserted after the `)`, can you remove it?

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
One small nitpick, LGTM for the rest

File: Assets/Scripts/Core/Controllers/GridCombatSystem.cs (hunk 1 above)
Comment (Westlanderz) [reply Assets/Scripts/Core/Controllers/GridCombatSystem.cs:122]: Strange vscode rendered it as an space but it had the unicode character for the tab there if you looked very closely. Should be fixed now.

**Review by Dennisbonke — APPROVED** [review]
LGTM now

File: Assets/Scripts/Core/Controllers/GridCombatSystem.cs (hunk 1 above)
Comment (Dennisbonke) [reply Assets/Scripts/Core/Controllers/GridCombatSystem.cs]: Yeah github rendered it funny and upon closer inspection it turned out to be a tab. It looks fixed now.

**Pull request merged**